jsonnet/main.jsonnet

```diff
@@ -0,0 +1,35 @@
+local excludedRules = [
+  {
+    name: 'etcd',
+    rules: [
+      { alert: 'etcdHighNumberOfFailedGRPCRequests' },
+      { alert: 'etcdInsufficientMembers' },
+    ],
+  },
+];
+
+local etcdMixin = (import 'github.com/etcd-io/etcd/contrib/mixin/mixin.libsonnet');
+local openshiftRules = (import 'custom.libsonnet');
+local a = if std.objectHasAll(etcdMixin, 'prometheusAlerts') then etcdMixin.prometheusAlerts.groups else [];
+local r = if std.objectHasAll(etcdMixin, 'prometheusRules') then etcdMixin.prometheusRules.groups else [];
+local o = if std.objectHasAll(openshiftRules, 'prometheusRules') then openshiftRules.prometheusRules.groups else [];
+local rules = std.map(function(group) group { rules: std.filter(function(rule) !std.setMember(rule.alert, ['etcdHighNumberOfFailedGRPCRequests', 'etcdInsufficientMembers']), super.rules) }
+                      , a + r);
+
+{
+  apiVersion: 'monitoring.coreos.com/v1',
+  kind: 'PrometheusRule',
+  metadata: {
+    name: 'etcd-prometheus-rules',
+    namespace: 'openshift-etcd-operator',
+    annotations:
+      {
+        'include.release.openshift.io/ibm-cloud-managed': 'true',
```

Comment:
I think there is a non-zero chance that developers external to the team may need to add more of these annotations in the future. Maybe inject a comment into the manifest after conversion to yaml (since json doesn't support comments) and also add a verify check that prevents a manual change from being committed (since it would be overwritten by a subsequent generate invocation)?

I'm guessing this is what @hexfusion meant by updating make, though, so it can be done in a follow-up.

Reply:
As discussed will do in a follow up, and sync with other folks on this.